docs/docusaurus/versioned_docs/version-0.17.23/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -175,7 +173,7 @@ Using the  <TechnicalTag relative="../../../" tag="data_context" text="Data Cont
 
 In the example, we have created a Data Source named `my_bigquery_datasource`, using the `add_or_update_sql` method and passing in a connection string.
 
-To configure the BigQuery Data Source, see [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery).
+To configure the BigQuery Data Source, see [How to connect to a BigQuery database](/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md).
```

Comment:
Starting the link with `/docs/` points it to the current version of docs. This should be changed to start with `/guides` I might have missed other instances of this in the review

Reply:
I've corrected this in the links that this PR edits.